packages/hash/api/src/graphql/typeDefs/entityType.typedef.ts

```diff
@@ -67,6 +67,23 @@ export const entityTypeTypedef = gql`
     OrgEmailInvitation
   }
 
+  type DestructuredSchemaProperty {
+    name: String!
+    content: JSONObject!
+  }
+
+  type DestructuredSchemaParent {
+    id: String!
```

Comment:
I think we should include `entityId` as an alias here, to be consistent with how we're defining the Entity GQL type for instance.